lib/ECMA262Regex.pm6

```diff
@@ -222,7 +222,11 @@ class ECMA262Regex::ToPerl6Regex {
     }
 
     method pattern-character($/) {
-        make ~$/;
+        if ~$/ eq '#' {
+            make "'#'";
+        } else {
+            make ~$/;
+        }
```

Comment:
Hm, I fear not only the `#` can be a problem. I think the safest bet that avoids all such fun with escaping is just to turn it into codepoints, like `make $/.Str.ords.map({ "\\x" ~ .base(16) }).join;`.

Reply:
Done.